docs/content/docs/connectors/models/openai.md

```diff
@@ -157,6 +157,45 @@ FROM ML_PREDICT(
                 </ul>
             </td>
         </tr>
+        <tr>
```

Comment:
We may want to describe how to use the metadata column to extract the error message and give an example.

Reply:
I have added a section to describe when to use metadata and their schemas. It's a little different from the "example" you mentioned, and I think the current document should be more straightforward.